sdk/sdk-view/src/test/java/mil/emp3/core/StorageManagerTest.java

```diff
@@ -0,0 +1,462 @@
+package mil.emp3.core;
```

Comment:
> Many things you sate above already have associated unit tests. 

I assume you mean the BasicAddOverlayAndFeatureTest class? It looks like it is attempting to cover a broader range of functionality than what is typically called for in unit test. I don't disagree that those types of tests need to be done, I'm simply stating that what the StorageManagerTest class aimed to achieve is unit testing only parts of the Storage/EventManager methods that accept userContext references. It still needs to be updated to test the other methods.

> The test we need is to make sure that filtering is actually happening when user specifies it. 

We aren't filtering anything. The ability is given to the user to recognize a userContext object and with that what they will, such as filtering. Here we are just testing that the userContext object is actually being returned back to us.

> Also what happens in a multi thread environment.

I think this is outside the scope of a unit test.